yb_migrate/cmd/exportDataStatus.go

```diff
@@ -73,8 +73,7 @@ func initializeExportTablePartitionMetadata(tableList []string) {
 		if source.DBType == ORACLE {
 			partitionList := migration.OracleGetAllPartitionNames(&source, parentTable)
 			if len(partitionList) > 0 {
-				msg := fmt.Sprintf("Table %q has partitions: %v\n", parentTable, partitionList)
-				utils.PrintAndLog(msg)
+				utils.PrintAndLog("Table %q has partitions: %v\n", parentTable, partitionList)
```

Comment:
No trailing `\n` in the log messages.
If required, fix the `PrintAndLog()` implementation to suffix the msg with a `\n` before giving it to the `Print` call.

Reply:
Yeah, I missed this one, will fix and merge with other comments